packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -606,15 +606,15 @@ - (BOOL)hasCameraRollPermission
 }
 
 
-- (NSString *)mapJSPresetToNativePreset:(NSNumber *)preset API_AVAILABLE(ios(11));
+- (NSString *)importExportPreset:(NSNumber *)preset API_AVAILABLE(ios(11));
 {
   static NSDictionary* presetsMap = nil;
   if (!presetsMap) {
     presetsMap = @{
```

Comment:
`presetsMap` - what presets?
`videoCompressionPresets`. I dunno about `Map` suffix in name 😛

Reply:
I am for the `Map` suffix, it's only purpose is to **map** preset number to a preset, so the name explains its purpose

Reply:
I withdraw from my comment here 😉 I see that this variable is static and available from this method context only so I'm 💯% okay with the way how it's now 😉